enclave_mgmt/workflow_template.yaml

```diff
@@ -23,12 +23,10 @@ spec:
           - "-cxe"
           - |
             mkdir -p /data/enclave-input/ay2022
-            mkdir -p /data/enclave-input/su2023
             mkdir -p /data/enclave-input/ay2023
             mkdir /data/enclave-output
             aws s3 cp --recursive s3://raise-data/algebra1/ay2022/enclave/ /data/enclave-input/ay2022/
-            aws s3 cp --recursive s3://raise-data/algebra1/su2023/enclave/ /data/enclave-input/su2023/
-            CSV_OUTPUT_DIR=/data/enclave-input/ay2023 compile-models raise-data algebra1/ay2023 raise-data events
+            CSV_OUTPUT_DIR=/data/enclave-input/ay2023 compile-models raise-data algebra1/ay2023 raise-data events --research-filter-bucket raise-data --research-filter-prefix algebra1/ay2023/automation/courses.csv
```

Comment:
Per findings during testing, we need to update the flags to use `_` instead of `-` (e.g. `--research_filter_bucket`)